packages/backend/src/peripherals/database/migrations/061_rename_tvl_tables.ts

```diff
@@ -0,0 +1,74 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server.
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+const changes = [
+  {
+    oldName: 'asset_balances',
+    newName: 'balances',
+    indexes: ['pkey'],
+  },
+  {
+    oldName: 'balance_status',
+    newName: 'balances_status',
+    indexes: ['pkey', 'config_hash_index', 'unix_timestamp_index'],
+  },
+  {
+    oldName: 'report_status',
+    newName: 'reports_status',
+    indexes: ['pkey', 'config_hash_index', 'unix_timestamp_index'],
+  },
+  {
+    oldName: 'aggregate_reports',
+    newName: 'reports_aggregated',
```

Comment:
Do we really want to switch that around? By that I mean, what is better:

- `aggregated_reports`
- `reports_aggregated`

Everywhere else it's the other way around, so the DB would be the only odd one out

Reply:
I agree